README.md

```diff
@@ -2,28 +2,133 @@
 
 Website: [https://codexappaa.herokuapp.com/](https://codexappaa.herokuapp.com/)
 
+## What is CodeX?
+
+CodeX is an online publishing platform, for members of the tech industry, inspired by [Medium.com](https://medium.com/). On it, you can write, read, and connect with others, on such topics as:
+- software engineering
+- computer science
+- data science
+- machine learning
+- and more.
+
+### Did You Know?
+
+The codex, a bound collection of handwritten contents on non-paper materials, is the historical ancestor of the modern book. Conveniently, for our purposes, its pronunciation and spelling imply e**x**change of information, and echo computer **code**, prominent features of innovation and technology.
+
 ## Technologies Used
 
 - JavaScript
 - Express
-- PUG
+- Pug
 - CSS
+- BCrypt
+- Heroku
+- Sequelize
+- PostgreSQL
 
 ## Features
 
-- Website home page with stories
-- User registration / login (with a demo account available)
-  - An authenticated home page with recommended stories and stories of users you follow
-- Specific story pages to view the story written by an author
-  - A comments section, where authenticated users can add comments to a story
-  - Like/unlike feature on the story and its comments that update live
-  - Follow/unfollow feature that updates live
-- A story creation page allowing authenciated users to post new stories
-  - Supports markdown in the body of the story
+### Home Page with Stories
+
+![Home Page](https://github.com/kvh8899/week13-project/blob/main/docs/images/screens-readme/homepage.png)
+
+### User-Authenticated Home Page
+
+![Home Page - User Authenticated](https://github.com/kvh8899/week13-project/blob/main/docs/images/screens-readme/homepage-user_auth.png)
+
+*The feed features stories from users that you follow, and recommended stories.*
+	
+### User Registration and Login
+
+![Login](https://github.com/kvh8899/week13-project/blob/main/docs/images/screens-readme/login.png)
+
+*Demo account also available*
+
+### Story Page
+
+![Story - comments popover](https://github.com/kvh8899/week13-project/blob/main/docs/images/screens-readme/story-comments_and_likes.png)
+
+- Comments section - authenticated users can add comments to a story
+- Follow / unfollow a user
+- Like / unlike
+	- Authenticated users can like or remove their like on the story
+	- Authenticated users can like or remove their like on comments
+- Live updates for comments, follows, and likes.
+
+### Story Creation Page
+
+![Story Creation](https://github.com/kvh8899/week13-project/blob/main/docs/images/screens-readme/story-edit_with_markdown.png)
+
+*Authenciated users can post new stories. Body text input supports Markdown*
+
+![Story Edit](https://github.com/kvh8899/week13-project/blob/main/docs/images/screens-readme/story-edit.png)
+
+*The authenticated user can also edit stories*
+
+### Bonus Features
```

Comment:
Another bonus feature we had was the markdown support for story pages. You can just take a screenshot of one of the story pages that has some noticeable styling to the body

Reply:
I hadn't migrated those dummy stories to my local machine yet. Would you mind sending me a screenshot?